test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -642,7 +642,7 @@ class test_uct_cm_sockaddr : public uct_test {
         remote_data = connect_args->remote_data;
         status      = connect_args->status;
 
-        if (status == UCS_ERR_REJECTED) {
+        if ((status == UCS_ERR_REJECTED) || (status == UCS_ERR_UNREACHABLE)) {
```

Comment:
why?

Reply:
because after the changes in this PR, user's reject is UCS_ERR_REJECTED and network reject is UCS_ERR_UNREACHABLE.

Reply:
then the test has to handle network reject reparately

Reply:
there is no actual handling. it just sets a flag

Reply:
it's handling and that flag is called "TEST_STATE_CLIENT_GOT_REJECT"

Reply:
you want to split this flag to "CLIENT_GOT_NETWORK_REJECT" and "CLIENT_GOT_USER_REJECT" ?

Reply:
just reject and unreachable